.gitignore

```diff
@@ -7,3 +7,4 @@ dist-ssr
 
 /data
 !/data/.gitkeep
+.vscode
```

Comment:
Please also add it to `.dockerignore` file

Reply:
done, thanks